META-INF/plugin.xml

```diff
@@ -620,6 +620,10 @@ We keep delivering funded results =)
         groupName="Code Style"                                    enabledByDefault="false" level="WEAK WARNING"
         implementationClass="com.kalessil.phpStorm.phpInspectionsEA.inspectors.codeSmell.ParameterDefaultValueIsNotNullInspector"/>
     <localInspection language="PHP" groupPath="PHP,Php Inspections (EA Extended)"
+        shortName="ParameterEqualsDefaultValueInspection"         displayName="Calls with the same default value"
```

Comment:
Would you please apply following changes: title = "Unnecessary arguments specification", group = "Unused".